app/models/user.rb

```diff
@@ -82,12 +82,18 @@ class User
   # email presence validated through Devise
   validates :name, presence: true
   validates :role, presence: true
-  validate :secure_password
+  validate :secure_password, if: :updating_password?
+  # i18n-tasks-use t('errors.messages.password.password_strength')
+  validates :password, password_strength: {use_dictionary: true}, if: :updating_password?
```

Comment:
Should we also add `min_word_length`? Docs say that the default is 4, I recommend setting it to 6 or 8.